### ISSUE TITLE
Update to the latest mastodon.py

### ISSUE THREAD

**Issue by magicalraccoon** [magicalraccoon/tootstream#115]
Hopefully this won't break too much.

**Comment by ng-0** [comment]
Any news on this one? It would be really great to get the changes for 1.1.0 Mastodon.py compatible to tootstream. The package I'm creating for guix can be used on every GNU-Linux system which can run Guix (and you could also make use of "guix pack" for others).

**Comment by craigmaloney** [comment]
Looks like there's a little update that needs to happen for Mastodon 1.1.0. I did a quick test and the "note" command no longer receives a string (and instead receives a datetime tuple). So there will need to either be a test to see what type is returned or a hard upgrade to Mastodon 1.1.0.

That's just one half-hour poking around. There may be others lurking in there.

(I did make one test where it did Stylize, but that didn't seem to return any date).
